### ISSUE TITLE
Regolith Wallpaper Collection

### ISSUE THREAD

**Issue by marauder316** [regolith-linux/regolith-desktop#617]
- [x] I've [run a search on the existing enhancements](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests) to make sure I'm not filing a duplicate (you can use the search box at the top left corner)
- [x] I've read [the Regolith documentation](https://regolith-linux.org/docs/https://regolith-linux.org/docs/) and didn't find any relevant topic
- [x] I tried [asking on Slack](https://regolith-linux.org/interact/) (especially for configuration related issues)

**Is your feature request related to a problem? Please describe.**
No, this is not related to a problem with Regolith.

**Describe the solution you'd like**
I was wondering if there was some repository on Github devoted to the wallpapers for Regolith, easily downloadable, sorted by look/theme, maybe with tags. I know where to find some wallpapers on the actual system, but I do not know that it contains a collection of _all_ of them, old and current.

**Describe alternatives you've considered**
Another alternative to a Github repo that makes sense could be a section of the website where the wallpapers (and their creators) are held.

**Additional context**
I'm aware of [regolith-assets](https://github.com/regolith-linux/regolith-assets), but that only has 4 of the wallpapers.

**Comment by cfsmp3** [comment]
Yes. Or well, coming soon. It's a pet project of mine. But it's not really a collection of wallpapers, rather is "wallpaper of the day" based on categories. 

If you want a good collection check out wall.alphacoders.com

I'm leveraging their API (with permission) for the wallpaper of the day stuff I'm getting ready.

**Comment by marauder316** [comment]
@cfsmp3 Cool, I'll try to stay tuned into the project. 😁

**Comment by marauder316** [comment]
@cfsmp3 I just had this thought:
Maybe for the wall of the day program you could set up [Pywal](https://github.com/dylanaraps/pywal) to do its thing. That way, not only would you have a new wallpaper but a theme to match as well. 🤔

User JollyRogerTrader had an unfinished project named [Regowal](https://github.com/JollyRogerTrader/Regowal). Last I tried it didn't work consistently. That could come in handy with this, unless I'm mistaken.
